WcaOnRails/app/models/user.rb

```diff
@@ -439,7 +439,7 @@ def can_edit_notes_of_user?(user)
   end
 
   def can_change_users_avatar?(user)
-    user.wca_id.present? && self.editable_fields_of_user(user).include?(:remove_avatar)
+    (user.wca_id.present? && self.editable_fields_of_user(user).include?(:remove_avatar)) || results_team?
```

Comment:
I don't think this will do quite what we want. In particular, we don't allow people to have an avatar if they don't have a WCA ID (not even a website admin is allowed to assign an avatar to someone if they don't have a WCA ID). I think the more correct fix would be to update `editable_fields_of_user` so it includes `:remove_avatar` if the use is a member of the WRT. That will mean updating this line of code https://github.com/jfly/worldcubeassociation.org/blob/9af3696f686a8baa6bbb6023e1ca3695a648c604/WcaOnRails/app/models/user.rb#L642, which will also give non delegate WRT members the power to edit a few other things as well, which we probably want them to have.